### ISSUE TITLE
Civic Tech Heroes of 2015 - Who should be on the list?

### ISSUE THREAD

**Issue by candacefaber** [openseattle/discuss#27]
Hey everyone - I'm writing a piece for Crosscut on the top civic tech projects from 2015. I want to make sure I haven't missed anyone/anything! I'll need to include detailed information about each. Featured projects should either have launched this year or achieved some sort of major milestone. Can be from within Open Seattle or people we know outside of this group who are working on civic projects. **If possible, I'd love a point of contact I can interview for a quick quote & to make sure what I write is accurate - including how people can get involved/provide additional support.**

Here's what I have so far:
- Seward Park Stories
- Duwamish Lighthouse (still need the anchor for 2015)
- Hey Duwamish! (grant from Tech Matching Fund)
- Access Map 
- Popup Arcade (Awesome Foundation grant and Seattle Mini Maker Fare)
- Seattle U. Social Justice Hackathon
- Mary's Place/SIC "Design Swarm"

What am I missing? Anything awesome that flew under the radar?

**Comment by sethvincent** [comment]
http://www.seattleghosts.com – not associated with open seattle. super cool though.

**Comment by candacefaber** [comment]
Thanks @sethvincent - that is awesome. @chrismetcalf @YonasBerhe others - anything come to mind?

**Comment by chrismetcalf** [comment]
Perhaps Bruce Blood or Michael Mattmiller from the city? And of course @candacefaber!

**Comment by georgerlee** [comment]
Candace, what a great article to do! Yeah! I am the artist director behind the lighthouse and worked with a team of other civic tech folks like Luke Swart, Colin Dietrich, and Alan Vezina. Colin did an amazing project putting GPS and terrain tracking kits on to bikes to map Seattle bike infrastructure that could be on this list. I have a video, media kit etc, let me know what you need, I am all yours. @candacefaber

**Comment by candacefaber** [comment]
Yeah! That is awesome, George. I'm going to be working on this over the
weekend - if you can send whatever you have to candace.faber@seattle.gov
(along with a good email/tel for you), I'll start with that. Thank you!

On Fri, Dec 18, 2015 at 10:20 AM, George Lee <notifications@github.com>
wrote:

> Candace, what a great article to do! Yeah! I am the artist director behind
> the lighthouse and worked with a team of other civic tech folks like Luke
> Swart, Colin Dietrich, and Alan Vezina. Colin did an amazing project
> putting GPS and terrain tracking kits on to bikes to map Seattle bike
> infrastructure that could be on this list. I have a video, media kit etc,
> let me know what you need, I am all yours. @candacefaber
> <https://github.com/candacefaber>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openseattle/discuss/issues/27#issuecomment-165862432>.
>



-- 

Candace Faber

Seattle, WA

twt: @candacefaber

candacefaber.com <http://www.candacefaber.com>

**Comment by georgerlee** [comment]
Hi Candace, just checking, did you get my email with materials I sent on Saturday morning? Cheers, George

**Comment by candacefaber** [comment]
Yes, I did! Thank you, George!
